### PR TITLE
Added run peer node notes

### PR DESCRIPTION
**Pull request by dgarcia360**
Related to #64

### CODE REVIEW

File: docs/RUNPEERLIN.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+and connect it to an existing network using the catapult-server build.
+
+Hardware requirements
+---
+The scripts have been tested on servers with the following minimum requirements.
+
+* CPU: 2 cores or more
+* Memory: 4GB or more
+* HD: 20GB or more
+* OS: Ubuntu 18.04 (LTS)
```
Comment (Jaguar0625) [inline docs/RUNPEERLIN.md:14]: can you add comment to effect that requirements are network dependent (i.e. networks with higher throughput will likely have higher requirements) ?

File: docs/RUNPEERLIN.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+---
+
+After building catapult-server, copy the configuration templates from the root folder of the repository under the ``_build`` directory.
+       
+```sh
+   cd catapult-server/_build
+   cp ../resources/* resources/
+```
+
+Replace the network configuration
```
Comment (Jaguar0625) [inline docs/RUNPEERLIN.md:36]: it is recommended to download all from running network. 
these configurations are not intended to be used by any "real" network

File: docs/RUNPEERLIN.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+``` ini
+[localnode]
+host = <YOUR_NODE_IP>
+friendlyName = myNode
+version = 0
+roles = Peer
+```
+
+2\. Open the ``config-user.properties`` and replace ``bootPrivateKey`` with the node's private key.
```
Comment (Jaguar0625) [inline docs/RUNPEERLIN.md:72]: this is no longer accurate with TLS changes and has been removed. please review NIP created by @gimer

File: docs/RUNPEERLIN.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    "metadata": {
+    "name": "peernode",
+    "roles": "Peer"
+    }
+    }
+    ]
+   }
+```
+
+NOTE: If the network has more peer nodes, you can list them by adding new entries to the ``knwonPeers`` array.
```
Comment (Jaguar0625) [inline docs/RUNPEERLIN.md:105]: `knownPeers`

Comment (Jaguar0625) [inline docs/RUNPEERLIN.md:105]: might want to mention that intent is for these nodes to serve as beacons, so they should be strong. for best network performance, a random subset of beacons should be set in each node's peer file.

File: docs/RUNPEERLIN.md (hunk 3 above)
Comment (dgarcia360) [reply docs/RUNPEERLIN.md]: Ok, will do but might take a while. I have to recompile catapult-server and play with the changes.

**Pull request merged**